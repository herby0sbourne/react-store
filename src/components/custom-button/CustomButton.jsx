import React from 'react';
import './custom-button.scss';

const CustomButton = ({ children, isGoogleSignIn, inverted, ...props }) => {
  return (
    <button
      // className={`${isGoogleSignIn && 'google-sign-in'} custom-button`}
      className={`${inverted ? 'inverted' : ''} ${
        isGoogleSignIn ? 'google-sign-in' : ''
      } custom-button`}
      {...props}
    >
      {children}
    </button>
  );
};

export default CustomButton;
