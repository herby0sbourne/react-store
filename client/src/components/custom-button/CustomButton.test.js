import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import CustomButton from './CustomButton.jsx';

test('should render CustomButton', () => {
  const wrapper = shallow(<CustomButton />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
