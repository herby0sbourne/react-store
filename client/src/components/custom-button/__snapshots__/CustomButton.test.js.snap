// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CustomButton 1`] = `<styled.button />`;
